cli/cmd/profile.go

```diff
@@ -94,6 +104,19 @@ Examples:
   If the --proto flag is specified, it reads the given protobuf definition file
   and outputs a corresponding service profile:
 
+  If the --tap flag is specified, it runs linkerd tap target for --tap-duration seconds,
+  and creates a profile for the SERVICE based on the requests seen in that window:
+
+  linkerd profile books --tap deploy/books --tap-duration 10s --tap-route-limit 5 > book-svc-profile.yaml
+  # (edit book-svc-profile.yaml manually)
+  kubectl apply -f book-svc-profile.yaml
+
+  The command will run linkerd tap deploy/books for tap-duration seconds, and then create
+  a service profile for the books service with routes prepopulated from the tap data.
+  For high RPS, high-route-cardinality services, use tap-route-limit to limit the number of
+  routes in the output profile.
+
+  linkerd profile -n emojivoto --open-api web-svc.swagger web-svc | kubectl apply -f -,
   linkerd profile -n emojivoto --proto Voting.proto vote-svc | kubectl apply -f -`,
```

Comment:
move this line up to right below `If the --proto flag...`

Reply:
good catch! okay I /think/ the examples section is ok now